pkg/arvo/sys/hoon.hoon

```diff
@@ -2133,7 +2133,13 @@
   ::    3f: scrambling                                  ::
   ::    3g: molds and mold builders                     ::
   ::                                                    ::
-~%  %tri  +  ~
+~%  %tri  +
+  ==
+    %year  year
+    %yore  yore
+    %fein  fein
+    %fynd  fynd
+  ==
```

Comment:
I don't think it's right to rearrange these cores for incrementally easier hooking. If you just declare `%ob` here, the hint `+ob` and declares hooks on it for `%fein` and `%fynd`, you can hook them in the jets in two steps: hook the core for `+ob`, then hook the product of that for `+fein` or `+fynd`.